aezeed/cipherseed_test.go

```diff
@@ -502,6 +571,45 @@ func TestDecipherIncorrectMnemonic(t *testing.T) {
 
 }
 
+// TestBirthdayCalculationGenesisDate tests that if the bitcoin genesis block time
+// is used, the birthday value of the seed is 0
+func TestBirthdayCalculationGenesisDate(t *testing.T) {
```

Comment:
It seems these two tests can be collapsed into the test vectors. Both of the vectors are already populated with the timestamps used in these tests, we just need to add an `expectedBirthday` field to the vectors and assert the condition.